Packs/ZeroDayLiveTIFusion/Integrations/ZeroDayLiveTIFUSION/Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
```

Comment:
Please remove the pipfile as we are looking at the docker image and do not use the pipfile.